codec-http/src/main/java/io/netty/handler/codec/http/multipart/HttpPostStandardRequestDecoder.java

```diff
@@ -783,6 +784,17 @@ public void removeHttpDataFromClean(InterfaceHttpData data) {
         factory.removeHttpDataFromClean(request, data);
     }
 
+    /**
+     * Check if request has headers indicating that it contains form body
+     */
+    private boolean hasFormBody() {
+        String contentHeader = request.headers().get("Content-Type");
```

Comment:
Use static HttpHeaderNames value

Reply:
fixed, also renamed the `contentHeader` variable to `contentHeaderValue` :)